src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -901,8 +899,15 @@ class ParseLiveQueryServer {
         subscription: subscription,
       };
       // Add selected fields, sessionToken and installationId for this subscription if necessary
+      if (request.query.keys) {
+        subscriptionInfo.keys = Array.isArray(request.query.keys) ? request.query.keys : [request.query.keys];
+      }
       if (request.query.fields) {
-        subscriptionInfo.fields = request.query.fields;
+        subscriptionInfo.keys = request.query.fields;
```

Comment:
Since this is not setup like `keys` above, it seems like it might take a comma separated string? I know some other clients will only send the comma separated string

Reply:
I think so, but I'm going to leave it as is for backwards compatibility